client/src/assets/Images/Spielplatzfinder-Dateien/leaflet.css

```diff
@@ -0,0 +1,640 @@
+/* required styles */

```

Comment:
Also this, this doesn't look like an image and i'm not sure if you need to have it locally?

Reply:
trying to fix asap.